hwdb.d/60-keyboard.hwdb

```diff
@@ -590,8 +590,9 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHewlett-Packard*:pnHDX9494NR:pvr*
 
 # HP EliteBook 725 G2
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHewlett-Packard*:pnHPLicrice:pvr*
-# HP EliteBook 840 G1
-evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHewlett-Packard*:pnHPEliteBook840G1:pvr*
+# HP EliteBook
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHewlett-Packard*:pnHPEliteBook*:pvr*
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHP*:pnHPEliteBook*:pvr*
```

Comment:
hmm, a few lines up there are other lines applied to certain EliteBook models. I wonder if it's really right to make these lines so generic, if there are apparently other EliteBook models for which this is not appropriate?

Reply:
Ok, let me double confirm with HP.

Reply:
HP confirms all EliteBooks use the common scan code.

Pushed a new one to merge other EliteBook entries.